examples/Example/src/combo/index.tsx

```diff
@@ -79,20 +96,32 @@ class TouchableHighlight extends Component {
   }
 }
 
-var INACTIVE_CHILD_STYLE = StyleSheet.create({ x: { opacity: 1.0 } }).x;
+var INACTIVE_CHILD_STYLE = StyleSheet.create({x: {opacity: 1.0}}).x;
 const INACTIVE_UNDERLAY_STYLE = StyleSheet.create({
-  x: { backgroundColor: 'transparent' },
+  // eslint-disable-next-line react-native/no-unused-styles
+  x: {backgroundColor: 'transparent'},
 }).x;
```

Comment:
Why we can't assign `backgroundColor` object direcly? This should be the same, right?

Reply:
We can check if it works, then why not.